app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/DetailActivity.java

```diff
@@ -25,11 +42,17 @@
 
     public static final int ANIMATION_DURATION = 500;
 
-    Person p;
+    private Person p;
```

Comment:
For the sake of readability, don't use very short (ie. one character) variable names for non-temporary variables. Since `p` is used a lot, change the variable name to something longer like `person`.